### ISSUE TITLE
VSCode icon

### ISSUE THREAD

**Issue by tonualexandru** [microsoft/vscode#120611]
Issue Type: <b>Bug</b>

After update to version 1.55.0 vscode icon is distortioned in context menu (open with) and same in Applicications folder.

OS version: 10.13.6 (17G14042)

VS Code version: Code 1.55.0 (c185983a683d14c396952dd432459097bc7f757f, 2021-03-30T16:01:05.981Z)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-2677M CPU @ 1.80GHz (4 x 1800)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>protected_video_decode: disabled_off<br>rasterization: disabled_software<br>skia_renderer: disabled_off_ok<br>video_decode: disabled_software<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|4, 3, 3|
|Memory (System)|4.00GB (0.03GB free)|
|Process Argv|--crash-reporter-id c493992a-5860-44ce-a2ea-ea137c8c61a7|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-intelephense-client|bme|1.6.3
vscode-eslint|dba|2.1.19
gitlens|eam|11.3.0
vscode-jest-runner|fir|0.4.31
shell-format|fox|7.0.1
gc-excelviewer|Gra|3.0.41
vscode-sshfs|Kel|1.20.0
sftp|lix|1.12.9
vscode-autoprefixer|mrm|3.0.1
python|ms-|2021.3.680753044
jupyter|ms-|2021.5.702919634
remote-ssh|ms-|0.65.1
remote-ssh-edit|ms-|0.65.1
cpptools|ms-|1.2.2
vscode-typescript-tslint-plugin|ms-|1.3.3
vscode-xml|red|0.15.0
vscode-scss-formatter|sib|2.1.0
code-spell-checker|str|1.10.2
sass-indented|syl|1.8.16
open-in-browser|tec|2.0.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspyt653:30270858
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30276681
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
pythonvsnew554:30281908
pythontb:30283811
wslfolderdoc:30282074
vspre833cf:30267465
pythonptprofiler:30281270
vshan820:30276952
pythondataviewer:30285071

```

</details>

**Comment by gjsjohnmurray** [comment]
/duplicate of #120305 and will be fixed in upcoming 1.55.1

**Comment by tonualexandru** [comment]
Screenshots:
Context menu:  https://imgur.com/kKFhQmh
Applications folder: https://imgur.com/f9yl0Z7
